ui-test/src/main/java/base/BasePage.java

```diff
@@ -229,4 +230,35 @@ public String getElementTagName(WebElement element) {
 		LOGGER.info("Retrieved text: {}", text);
 		return text;
 	}
+
+	public String authorizeUrl;
+
+	public String getAuthorizeUrl() {
+		return authorizeUrl;
+	}
+
+	public void setAuthorizeUrl(String url) {
+		this.authorizeUrl = url;
+	}
+
+	public String getExpectedDefaultLanguage() {
+		String url = getAuthorizeUrl();
+
+		if (url == null || url.isEmpty()) {
+			url = driver.getCurrentUrl();
+			setAuthorizeUrl(url);
+		}
+		return ClaimsUtil.mapLangToName(ClaimsUtil.getDefaultLanguageFromUrl(url));
+	}
+
+	public String getExpectedFullNameInKhmerPlaceholder(String lang) {
```

Comment:
For 1.7.0 we are targetting configurable registration form in the signup UI @mohanachandran-s 
So i think in all the places where we have considered to enter full name in khmer it should be made dynamic based on the UI spec.